README.md

```diff
@@ -1,71 +1,108 @@
-# Puppet-lint
+# Puppet Lint
 
 [![Build
 Status](https://secure.travis-ci.org/rodjek/puppet-lint.png)](http://travis-ci.org/rodjek/puppet-lint)
 [![Inline docs](http://inch-ci.org/github/rodjek/puppet-lint.png?branch=master)](http://inch-ci.org/github/rodjek/puppet-lint)
 
-Puppet Lint will test modules and manifests against the recommended Puppet
-style guidelines from the [Puppet Labs style
-guide](http://docs.puppetlabs.com/guides/style_guide.html) as closely as practical. It is not meant to validate syntax. Please use `puppet parser validate` for that.
+Puppet Lint tests modules and manifests against the recommended Puppet style guidelines. It can also help automatically fix style issues in your code.
```

Comment:
Yep, I mashed them up and shortened it, but definitely keeping the bit about "not for syntax."